wp-content/themes/core/components/routes/not_found/Not_Found_Controller.php

```diff
@@ -71,11 +72,19 @@ public function render_breadcrumbs(): void {
 	 * @return \Tribe\Project\Templates\Models\Breadcrumb[]
 	 */
 	protected function get_breadcrumbs(): array {
-		$page = get_option( 'page_for_posts' );
-		$url  = $page ? get_permalink( $page ) : home_url();
+		return [
+			new Breadcrumb( '', __( 'Not Found', 'tribe' ) ),
+		];
+	}
 
+	/**
+	 * @return array
+	 */
+	public function get_search_form_args(): array {
 		return [
-			new Breadcrumb( $url, __( 'News', 'tribe' ) ),
+			Search_Form_Controller::CLASSES     => [ 'c-search--full', 'search-results__form' ],
```

Comment:
Totally just curious about these two particular classes. Does adding them implement existing styles for variations on the search form?